observation_portal/proposals/tests.py

```diff
@@ -295,3 +310,15 @@ def test_activate_selected(self):
         )
         for proposal in self.proposals:
             self.assertEqual(Proposal.objects.get(pk=proposal.id).active, True)
+
+    def test_clean_time_allocation(self):
+        ta_dict = self.time_allocation.as_dict()
+        taf = TimeAllocationForm(data=ta_dict, instance=self.time_allocation)
+        self.assertTrue(taf.is_valid())
+
+        # Now attempt to change the semester and see it fail
```

Comment:
I tried to add this unit test. It works in isolation, but if I run it uncommented with all other unit tests, it breaks _other_ unrelated unit tests! I've traced it to the TimeAllocationForm with data with the changed semester id, but have no idea why this break other tests?

Reply:
Interesting, nothing is standing out to me about the test that would break other tests.

Reply:
yeah... it is pretty frustrating since it doesn't really have any clear reason why it would impact other tests. Its basically just calling form validation which shouldn't make any changes